src/main/kotlin/gg/ingot/iron/annotations/Column.kt

```diff
@@ -1,9 +1,14 @@
 package gg.ingot.iron.annotations
 
+import gg.ingot.iron.serialization.ColumnDeserializer
+import gg.ingot.iron.serialization.EmptyDeserializer
+import kotlin.reflect.KClass
+
 @Target(AnnotationTarget.PROPERTY)
 @Retention(AnnotationRetention.RUNTIME)
 annotation class Column(
     val name: String = "",
     val ignore: Boolean = false,
-    val json: Boolean = false
-)
+    val json: Boolean = false,
+    val deserializer: KClass<out ColumnDeserializer<*, *>> = EmptyDeserializer::class
```

Comment:
We might want to add a type generic on this alongside the class to ensure full type safety

Reply:
Invalid request.